tests/integration/suite/daprd/pubsub/http/fuzz.go

```diff
@@ -19,7 +19,7 @@ import (
 	"crypto/rand"
 	"encoding/base64"
 	"encoding/json"
-	"errors"
+	stderrors "errors"
```

Comment:
is this change so the focus for `errors` tests can be consistent across all apis? Ik its overlapping the names

Reply:
Yeah, I think the three of us talked about this and had decided that we keep the std packages with their own names and import the kit error under the same alias everywhere. I'm fine with whatever decision, as long as it's consistent.

Reply:
:+1: